egs/chime1/s5/local/chime1_prepare_data.sh

```diff
@@ -53,9 +52,8 @@ for x in "devel" "test"; do
     wav_dir="${!wav_var}"
     for sid in `seq 34`; do
       sid2=`printf "s%02d" $sid`
-      tab=$'\t'
       ls -1 $wav_dir/*/s${sid}_*.wav \
-        | sed "s/\(.*\)\/\(.*\)\/s.*_\(.*\).wav/${sid2}_\3_\2${tab}\1\/\2\/s${sid}_\3.wav/" \
+        | perl -ape "s/(.*)\/(.*)\/s.*_(.*).wav/${sid2}_\3_\2$\t\1\/\2\/s${sid}_\3.wav/;" \
```

Comment:
again, for complex things like this you should either test somehow, or leave the sed command.

Reply:
yep. I actually tested every complex perl command I wrote...  The major feature not compatible I've encountered that is that "(" is already a special symbol in perl so we don't need the backslash before it.